docs/documentation/quartz-3.x/db/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Database Schema
+---
+
+When using the JobStoreTX ADO, Quartz.Net will create 11 tables.
```

Comment:
Just checking, I don't think Quartz creates tables, the user needs to run the scripts manually. 
Perhaps something like 


"When using JobStoreTX ADO, Quartz requires the creation of a set of tables.  This is a manual step, as Quartz.Net does not create these automatically.  These scripts can be found in the repo [database/tables](https://github.com/quartznet/quartznet/tree/main/database/tables).

Reply:
I tweaked the wording a bit.